COMMERCE-9634 - No warning message is shown when creating a product without back order or units in inventory

victorcdp opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

katienesterovich commented:
ci:test:sf

katienesterovich commented:
ci:test:relevant

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-21.liferay.com/job/test-portal-source-format/2719/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-21.liferay.com//userContent/jobs/test-portal-source-format/builds/2719/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 2 minutes 16 seconds 678 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/22cde9ad7563e8696feb1e27b1c1c6367098ddfb">22cde9ad7563e8696feb1e27b1c1c6367098ddfb</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/katienesterovich/liferay-portal/tree/LPS-103270-2">LPS-103270-2</a><br/>Branch GIT ID: <a href="https://github.com/katienesterovich/liferay-portal/commit/981de4e8ad5776c118b037bef233764a685d61a9">981de4e8ad5776c118b037bef233764a685d61a9</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-source-format/2719/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-source-format/builds/2719/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest(master)/4900/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-21.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4900/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 15 out of 26 jobs passed in 1 hour 36 minutes 21 seconds 656 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/1fc2e6c16fa20948ef1e8a8af68efcc85b7f6541">1fc2e6c16fa20948ef1e8a8af68efcc85b7f6541</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/6e88af622623004cef5307d748716e8007678148">6e88af622623004cef5307d748716e8007678148</a></p>15 out of 26 jobs PASSED<div><h4>11 Failed Jobs:</h4><ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest(master)/4900/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/329766/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</strong></strike></a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/337582/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326548/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326547/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</strong></strike></a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/337581/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326546/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</strong></strike></a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201459/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/js-unit-private-jdk8</strong></strike></a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201461/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</strong></strike></a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201458/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</strong></strike></a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201460/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</strong></strike></a></li></ul></div><details><summary><strong>15 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/349123/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk11_open/0</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299281/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201464/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299278/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/329767/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299277/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/329768/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201465/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326549/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201463/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299279/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/329770/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201462/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/329769/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299280/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-21.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4900/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/201461/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/201461//consoleText">AXIS_VARIABLE=0,label_exp=!master #201461</a><pre><code>[beanshell] github.upstream.branch.name=master
[beanshell] github.sender.branch.sha=981de4e8ad5776c118b037bef233764a685d61a9
[beanshell] github.receiver.username=brianikim
[beanshell] github.sender.branch.name=LPS-103270-2
[beanshell] github.pull.request.number=66
[beanshell] github.upstream.branch.sha=1fc2e6c16fa20948ef1e8a8af68efcc85b7f6541
[beanshell] github.sender.username=katienesterovich
[beanshell] local.git.branch.cached=true
[beanshell] github.origin.name=katienesterovich
[beanshell] 
     [echo] WARNING: lpkg-base-jdk8 does not match existing test types (blade-samples,database-upgrade-client,functional,jsf,plugins,portal-startup,service-builder,tck,modules-integration,integration,modules-unit,modules-unit-project-templates,unit)
     [echo] WARNING: lpkg-base-jdk8 does not match existing test types (blade-samples,database-upgrade-client,functional,jsf,plugins,portal-startup,service-builder,tck,modules-integration,integration,modules-unit,modules-unit-project-templates,unit)
     [echo] record.testray.results.start.timestamp: 10-21-2019 11:01:24:730 PDT
[stopwatch] [record.testray.results: 0.000 sec]

BUILD FAILED
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:11097: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common-portal-acceptance-batch.xml:218: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:8417: Sourced file: inline evaluation of: `` 								 									import com.liferay.jenkins.results.parser.GitHubDevSyncUtil;  . . . '' : Typed variable declaration : Typed variable declaration : Method Invocation GitHubDevSyncUtil.fetchCacheBranchFromGitHubDev : at Line: 47 : in file: inline evaluation of: `` 								 									import com.liferay.jenkins.results.parser.GitHubDevSyncUtil;  . . . '' : GitHubDevSyncUtil .fetchCacheBranchFromGitHubDev ( gitWorkingDirectory , remoteCacheBranchName ) 

Called from method: fetchCacheBranchToLocalBranch : at Line: 112 : in file: inline evaluation of: `` 								 									import com.liferay.jenkins.results.parser.GitHubDevSyncUtil;  . . . '' : fetchCacheBranchToLocalBranch ( getCacheBranchName ( ) , project .getProperty ( "top.level.branch.name" ) ) 
Target exception: java.lang.RuntimeException: Unable to fetch cache-brianikim-1fc2e6c16fa20948ef1e8a8af68efcc85b7f6541-katienesterovich-981de4e8ad5776c118b037bef233764a685d61a9 from cloud-10-0-0-37</code></pre></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">acceptance upstream results</a> at 1fc2e6c16fa20948ef1e8a8af68efcc85b7f6541:</strong></summary><ol><li><div><h5><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/329766/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></h5><div><h6>Job Results:</h6><p>6 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/329766//consoleText">AXIS_VARIABLE=0,label_exp=!master #329766</a><ol><li><div><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/329766//testReport/com.liferay.portal.modules/ModulesStructureTest/testScanBuildScripts">ModulesStructureTest.testScanBuildScripts</a><pre><code>junit.framework.AssertionFailedError: Forbidden modules/dxp/apps/analytics/build.gradle
	at com.liferay.portal.modules.ModulesStructureTest$1.preVisitDirectory(ModulesStructureTest.java:200)
	at com.liferay.portal.modules.ModulesStructureTest$1.preVisitDirectory(ModulesStructureTest.java:121)
	at java.nio.file.Files.walkFileTree(Files.java:2677)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.modules.ModulesStructureTest.testScanBuildScripts(ModulesStructureTest.java:119)
</code></pre></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/337582/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>12 Tests Passed.<br/>15 Tests Failed.</p></div><ol><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/337582//consoleText">AXIS_VARIABLE=0,label_exp=!master #337582</a><ol><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPages_AddContentPage_">LocalFile.ContentPages#AddContentPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentPages_AddContentPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentPages_AddContentPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPages_AddContentPageFromPageTemplate_">LocalFile.ContentPages#AddContentPageFromPageTemplate</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentPages_AddContentPageFromPageTemplate/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.ContentPages_AddContentPageFromPageTemplate/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/337582//consoleText">AXIS_VARIABLE=1,label_exp=!master #337582</a><ol><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPages_AddContentPageWithOOTBFragment_">LocalFile.ContentPages#AddContentPageWithOOTBFragment</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentPages_AddContentPageWithOOTBFragment/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentPages_AddContentPageWithOOTBFragment/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPages_DiscardContentPageDraft_">LocalFile.ContentPages#DiscardContentPageDraft</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentPages_DiscardContentPageDraft/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentPages_DiscardContentPageDraft/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPages_EditContentPage_">LocalFile.ContentPages#EditContentPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentPages_EditContentPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/LocalFile.ContentPages_EditContentPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/1/jenkins-console.txt.gz">Console Output</a></div></li><li>...</li></ol></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/337582//consoleText">AXIS_VARIABLE=2,label_exp=!master #337582</a><ol><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithEditables_AddContentPageWithImageFragment_">LocalFile.ContentPagesWithEditables#AddContentPageWithImageFragment</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/LocalFile.ContentPagesWithEditables_AddContentPageWithImageFragment/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/LocalFile.ContentPagesWithEditables_AddContentPageWithImageFragment/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithFragmentConfiguration_AddCheckboxConfigurationFragmentToContentPage_">LocalFile.ContentPagesWithFragmentConfiguration#AddCheckboxConfigurationFragmentToContentPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/LocalFile.ContentPagesWithFragmentConfiguration_AddCheckboxConfigurationFragmentToContentPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/LocalFile.ContentPagesWithFragmentConfiguration_AddCheckboxConfigurationFragmentToContentPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=2,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithFragmentConfiguration_AddTextConfigurationFragmentToContentPage_">LocalFile.ContentPagesWithFragmentConfiguration#AddTextConfigurationFragmentToContentPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/LocalFile.ContentPagesWithFragmentConfiguration_AddTextConfigurationFragmentToContentPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/LocalFile.ContentPagesWithFragmentConfiguration_AddTextConfigurationFragmentToContentPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/2/jenkins-console.txt.gz">Console Output</a></div></li><li>...</li></ol></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/337582//consoleText">AXIS_VARIABLE=3,label_exp=!master #337582</a><ol><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithSearch_FilterByPageFacet_">LocalFile.ContentPagesWithSearch#FilterByPageFacet</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/3/LocalFile.ContentPagesWithSearch_FilterByPageFacet/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/3/LocalFile.ContentPagesWithSearch_FilterByPageFacet/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/3/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=3,label_exp=!master/337582//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithSearch_SearchEditableText_">LocalFile.ContentPagesWithSearch#SearchEditableText</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/3/LocalFile.ContentPagesWithSearch_SearchEditableText/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/3/LocalFile.ContentPagesWithSearch_SearchEditableText/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/0/3/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326548/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></h5><div><h6>Job Results:</h6><p>22 Tests Passed.<br/>4 Tests Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/326548//consoleText">AXIS_VARIABLE=0,label_exp=!master #326548</a><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/326548//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithSearch_SearchStaticText_">LocalFile.ContentPagesWithSearch#SearchStaticText</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.ContentPagesWithSearch_SearchStaticText/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.ContentPagesWithSearch_SearchStaticText/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/326548//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithStaging_PublishContentPage_">LocalFile.ContentPagesWithStaging#PublishContentPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.ContentPagesWithStaging_PublishContentPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.ContentPagesWithStaging_PublishContentPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/jenkins-console.txt.gz">Console Output</a></div></li><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/326548//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_ContentPagesWithStagingAndPageTemplates_PublishContentPage_">LocalFile.ContentPagesWithStagingAndPageTemplates#PublishContentPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.ContentPagesWithStagingAndPageTemplates_PublishContentPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/LocalFile.ContentPagesWithStagingAndPageTemplates_PublishContentPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/326548//consoleText">AXIS_VARIABLE=1,label_exp=!master #326548</a><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/326548//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage_">LocalFile.DisplayPagesWithStagingAndWebContent#PublishEditedDisplayPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/1/LocalFile.DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/1/LocalFile.DisplayPagesWithStagingAndWebContent_PublishEditedDisplayPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/1/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/326547/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/326547//consoleText">AXIS_VARIABLE=0,label_exp=!master #326547</a><ol><li><div><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/326547//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_FragmentsUI_AddFragmentToContentPage_">LocalFile.FragmentsUI#AddFragmentToContentPage</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/3/0/LocalFile.FragmentsUI_AddFragmentToContentPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/3/0/LocalFile.FragmentsUI_AddFragmentToContentPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-21/1571678100277/test-portal-acceptance-pullrequest(master)/4900/functional-tomcat90-mysql57-jdk8/3/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li>...</li></ol></details></details></html>

brianikim commented:
Reviewing now :)

brianikim commented:
See: https://github.com/SamZiemer/liferay-portal/pull/677

Commit:
LRAC-12443 Add WIZARD_STEP_BUTTON path to navigate between wizard steps

Commit:
LRAC-12443 Update disconnectDXPFromAnalyticsCloud macro with the new paths

Commit:
LRAC-12443 Remove assert in success alert that is no longer used

Commit:
LRAC-12443 Update assertSuccessfulConnection macro

Commit:
LRAC-12443 Add NavigateBetweenWizardSteps macro

Commit:
LRAC-12443 Update error alert message

Commit:
LRAC-12438 Move CheckConnectionSuccessMessage to ConnectACWithDXP.testcase and rename the test to CheckIsPossibleConnectACWithValidToken

Commit:
LRAC-12440 Move InvalidToken to ConnectACWithDXP.testcase and rename the test to CheckUnableConnectACWithInvalidToken

Commit:
LRAC-12437 Add CheckConnectionEmptyStateMessage testcase

Commit:
LRAC-12438 Move ConnectACtoDXP to ConnectACWithDXP.testcase and rename the test to ConnectACToDXPAndSyncSitesAndContacts

Commit:
LRAC-12437 SF

Commit:
LRAC-12439 Add CheckDataSourceStatusUpdateWhenSyncSitesAndContacts testcase

Commit:
LRAC-12457 Update the paths to be able to navigate in the AC connection sidebar

Commit:
LRAC-12457 Update the macro to always use the newest data source that was created in the connection

Commit:
LRAC-12457 Add clickToAssignSitesAndChannel macro

Commit:
LRAC-12457 Add switchTab macro

Commit:
LRAC-12457 Add syncChannel macro

Commit:
LRAC-12457 Add syncSites macro

Commit:
LRAC-12457 Add NEW_PROPERTY_NAME path

Commit:
LRAC-12457 Update SYNCED_SITES_SEARCH_FIELD path

Commit:
LRAC-12457 Update AC_GENERIC_CHECKBOX path

Commit:
LRAC-12457 Remove unused path

Commit:
LRAC-12457 Remove unused macro

Commit:
LRAC-12457 Update some test with the changes made in the macros

Commit:
LRAC-12439 Add checkDataSourceStatus macro

Commit:
LRAC-12458 Add syncContactsAndAccountstData macro to sync contacts and accounts

Commit:
LRAC-12458 Add syncAccountsData macro to sync accounts and accountGroup

Commit:
LRAC-12458 Add syncContactsData macro to sync contacts, userGroup and organizations

Commit:
LRAC-12458 Update syncNewContactData macro

Commit:
LRAC-12458 Update SYNC_BY_ORGANIZATIONS_PREVIEW path

Commit:
LRAC-12458 Update SYNC_BY_USER_GROUPS_PREVIEW path

Commit:
LRAC-12457 Remove the tests because the new connection wizard doesn't have the functionality to create several properties at the same time

Commit:
LRAC-12458 Update connectDXPtoAnalyticsCloud macro with properties and contact synchronization changes

Commit:
LRAC-12458 Update some test with the changes made in the macros

Commit:
LRAC-12458 Ignore AssertLiferayDataSourceContactCountPreview because the new connection wizard since it does not have an indication of the number of users synchronized

Commit:
LRAC-12458 SF

Commit:
LRAC-12458 Update syncNewUserData macro

Commit:
LRAC-12458 Update sync macros with check if toggle is enabled

Commit:
LRAC-12457 Add createNewProperty macro

Commit:
LRAC-12458 Fix FIRST_AVAILABLE_PROPERTY path

Commit:
LRAC-12458 Fix connectDXPtoAnalyticsCloud macro

Commit:
LRAC-12570 Add CheckIfPropertyCreatedAutomaticallyAfterConnection testcase

Commit:
LRAC-12571 Add viewGenericListItem macro

Commit:
LRAC-12571 Add CanPutAnyNameInNewProperty testcase

Commit:
LRAC-12583 Fix createNewProperty macro

Commit:
LRAC-12583 Add CheckPropertyListSortedByCreationDateByDefault testcase

Commit:
LRAC-12583 SF

Commit:
LRAC-12584 Update createNewProperty macro to use for loop

Commit:
LRAC-12584 Update viewGenericListItem macro to use for loop

Commit:
LRAC-12584 Remove assert from an action macro

Commit:
LRAC-12584 Add CanPaginatePropertyListInDXP testcase

Commit:
LRAC-12585 Add viewGenericListItemNotPresent macro

Commit:
LRAC-12585 Add CanSearchForPropertiesInDXP testcase

Commit:
LRAC-12586 Add viewGenericListItemInOrder macro

Commit:
LRAC-12586 Add setSortDirection macro

Commit:
LRAC-12586 Add CanSortPropertyListInDXP testcase

Commit:
LRAC-12586 Remove the test that is covered by CanSearchForPropertiesInDXP testcase

Commit:
LRAC-12586 Remove the test that is covered by CanSearchForPropertiesInDXP testcase

Commit:
LRAC-12586 SF

manuferreira committed:
LRAC-12518 Add DisconnectACAndDXP file

Commit:
LRAC-12572 Update clickToAssignSitesAndChannel macro to assign sites and channel to a property passed as a parameter and update the tests

Commit:
LRAC-12572 Update syncChannel and syncSites macro with new variable to create property

Commit:
LRAC-12572 Remove assignSiteToExistingProperty macro

Commit:
LRAC-12572 Add CheckDeletedPropertyInACSideNotAppearInDXP testcase

Commit:
LRAC-12572 Remove the test that is covered by CheckDeletedPropertyInACSideNotAppearInDXP testcase

Commit:
LRAC-12572 Move OnlyAvailablePropertiesAppearInDXP testcase to ConnectionPropertyList.testcase file

Commit:
LRAC-12572 SF

Commit:
LRAC-12571 Update CanPutAnyNameInNewProperty testcase to cover the LRAC-12573

manuferreira committed:
LRAC-12518 Add xpath for when the value of the input token is empty

manuferreira committed:
LRAC-12518 Add DisconnectACAndDXPFromDXPSide testcase

manuferreira committed:
LRAC-12518 SF

Commit:
LRAC-12574 Add viewSyncedChannelsAndSitesNumber macro

Commit:
LRAC-12574 Add SelectedSitesAndChannelsNumbersCorrectAfterSync testcase

Commit:
LRAC-12575 Add SelectedSiteNumberCorrectWhenCommerceToggleDisabled testcase

Commit:
LRAC-12576 Update syncChannel macro to enable toggle only for the requested property

Commit:
LRAC-12576 Add CheckIsPossibleEnableToggleCommerceForTwoDifferentProperties testcase

Commit:
LRAC-12577 Add CHECKBOX_NOT_AVAILABLE_TO_SELECT path

Commit:
LRAC-12577 Add CheckSiteAndChannelOnlyBeSelectedForSingleProperty testcase

Commit:
LRAC-12578 Update the macro to use the uncheckToggleSwitch function to disable toggle

Commit:
LRAC-12578 Add CheckChannelListCannotEditedIfToggleDisabled testcase

Commit:
LRAC-12578 SF

Commit:
LRAC-12579 Add viewAssignModalResultsMessage macro

Commit:
LRAC-12579 Add CanSearchForChannelsAndSitesOnAssignedPropertyModal testcase

Commit:
LRAC-12580 Add changeAssignModalPagination macro

Commit:
LRAC-12580 Add ASSING_MODAL_NEXT_PAGE path

Commit:
LRAC-12580 Remove a parameter not needed by the macro

Commit:
LRAC-12580 Update the tearDown to remove all sites created

Commit:
LRAC-12580 Add MakePaginationInChannelsAndSitesTabsOnAssignedPropertyModal testcase

Commit:
LRAC-12581 Add setSortDirectionInAssignModal macro

Commit:
LRAC-12581 Add setFilterAndOrderInAssignModal macro

Commit:
LRAC-12581 Add viewAssignModalListItemInOrder macro

Commit:
LRAC-12581 Add SortInChannelsAndSitesTabOnAssignedPropertyModal testcase

Commit:
LRAC-12569 Fix createWorkspace macro

Commit:
LRAC-12569 Add CheckPropertyListEmptyStateMessage testcase

Commit:
LRAC-12569 SF

manuferreira committed:
LRAC-12658 Add testcase file

Commit:
LRAC-12679 Add clickToSelectAttributes macro

Commit:
LRAC-12679 Add syncAttributes macro

Commit:
LRAC-12679 Add desyncAttributes macro

Commit:
LRAC-12679 Add navToSyncedAttributes macro

Commit:
LRAC-12679 Update connectDXPtoAnalyticsCloud macro to finished the wizard connection

Commit:
LRAC-12679 Remove syncNewUserData macro

Commit:
LRAC-12679 Change the tests to use syncAttributes macro to sync a new attribute like custom field

Commit:
LRAC-12679 SF

Commit:
LRAC-12676 Add CanModifySitesAndChannelsSyncInPropertyReviewSidebar testcase

Commit:
LRAC-12677 Update SYNC_TOGGLE_ICON path

Commit:
LRAC-12677 Add SYNC_NUMBER_PREVIEW path

Commit:
LRAC-12677 Add viewSyncNumberPreview macro

Commit:
LRAC-12677 Fix syncAccountsData macro

Commit:
LRAC-12677 Fix syncContactsData macro

Commit:
LRAC-12677 Fix syncContactsAndAccountstData macro to disable toggle

Commit:
LRAC-12677 Add CanModifyContactsSyncInPeopleReviewSidebar testcase

Commit:
LRAC-12678 Rename searchACItem macro to searchACItemInAssignModal macro because it only works in assign modal

Commit:
LRAC-12678 Update macros to use searchBar macro for searching

Commit:
LRAC-12678 Add CanModifySelectedAttributesInAttributeReviewSidebar testcase

Commit:
LRAC-12678 Remove CannotAssignSiteToMultipleProperties testcase because it is covered by CheckSiteAndChannelOnlyBeSelectedForSingleProperty testcase

Commit:
LRAC-12678 Remove AssignSiteToExistingProperty testcase

Commit:
LRAC-12678 SF

Commit:
LRAC-12678 Move ConnectionStatusNotChangeWhenRenameDataSource from AnalyticsCloudConnectionTest to ConnectACWithDXP

manuferreira committed:
LRAC-12658 Add xpath to track the toggle button

manuferreira committed:
LRAC-12658 Add macro for commerce teadown

manuferreira committed:
LRAC-12658 Add macro to see the toggle on or off

manuferreira committed:
LRAC-12658 Add setup and teardown

manuferreira committed:
LRAC-12658 Add CanSyncAllContactsAndAccounts testcase

manuferreira committed:
LRAC-12658 CanSyncAllContacts testacase

manuferreira committed:
LRAC-12658 CanSyncAllAccounts testcase

manuferreira committed:
LRAC-12658 Add CanSyncByAccountGroups testcase

manuferreira committed:
LRAC-12658 Add CanSyncAllContactsByGroups testcase

manuferreira committed:
LRAC-12658 CanSyncAllContactsByOrganizations testcase

manuferreira committed:
LRAC-12658 CanSyncAllContactsAndAccountsGroupsOnly testcase

manuferreira committed:
LRAC-12658 Add CanSearchOrganizationsInContactsModal testcase

manuferreira committed:
LRAC-12658 Add CanSearchAccountGroupsInContactsModal testcase

manuferreira committed:
LRAC-12658 Add CheckIsPossiblePaginateOrganizations testcase

manuferreira committed:
LRAC-12658 Add CheckIsPossiblePaginateInAccountGroups testcase

manuferreira committed:
LRAC-12658 SF

Commit:
LRAC-12664 Add CanSearchGroupsInContactsModal testcase

Commit:
LRAC-12664 Add CheckIsPossiblePaginateGroups testcase

Commit:
LRAC-12664 Remove the test that is covered by CanSyncAllContactsByOrganizations

Commit:
LRAC-12664 Remove the test that is covered by CanSyncAllContactsByGroups

Commit:
LRAC-12664 Put the search tests to check the number of page results

Commit:
LRAC-12664 Add a comment to update the test when the commerce stories are finished

Commit:
LRAC-12664 SF

Commit:
LRAC-12664 Fix typing problem

robertoDiaz committed:
LPS-176235 Pass the className and ClassPK to check permissions

robertoDiaz committed:
LPS-176235 Add upgrade process to update all existing file entries and add their className and classPK

Commit:
LRAC-12762 Add CheckCanSortByOrganizationsName testcase

Commit:
LRAC-12763 Add CheckCanSortByUserGroupsName testcase

Commit:
LRAC-12764 Add CheckCanSortByAccountGroupsName testcase

Commit:
LRAC-12764 Fix tearDownCP macro

Commit:
LRAC-12764 SF

Commit:
LRAC-12777 Update macros to wait for loading icon

Commit:
LRAC-12777 Add getSyncNumberPreview macro

Commit:
LRAC-12777 Add viewIncreaseOrDecreaseSyncNumber macro

Commit:
LRAC-12777 Update the test to not use fixed numbers

Commit:
LRAC-12777 Add CanSelectAttributesForPeople testcase

Commit:
LRAC-12780 Add CanDeselectNoRequiredAttributesForPeople testcase

Commit:
LRAC-12781 Add CanSearchAttributeForPeopleList testcase

Commit:
LRAC-12782 Add CanPaginationAttributeForPeopleList testcase

Commit:
LRAC-12777 SF

joshitagaki committed:
LRAC-12302 Test fix wait for button to be enabled

manuferreira committed:
LRAC-12778 Add xpath to track the disabled attributes fields

manuferreira committed:
LRAC-12778 Add CheckNotPossibleToDeselectRequiredAttributesForPeople testcase

manuferreira committed:
LRAC-12778 Add CheckAllDefaultsAttributesAreCheckedForPeopleAttribute testcase

manuferreira committed:
LRAC-12778 Add CanSortAttributeForPeopleList testcase

Commit:
LRAC-12778 Fix Poshi syntax issues

Commit:
LRAC-12778 SF

calvinctvo committed:
LRAC-12842 Add search, sort, pagination test case for account attributes

calvinctvo committed:
LRAC-12842 SF

calvinctvo committed:
LRAC-12842 Change unsearch attribute

Commit:
LRAC-12901 Add REQUIRED_ATTRIBUTES path

Commit:
LRAC-12901 Add AllProductAttributesRequired testcase

Commit:
LRAC-12903 Add CanSearchAttributeInProductList testcase

Commit:
LRAC-12904 Add CanPaginateInProductAttributeList testcase

Commit:
LRAC-12905 Updating the path so it can be used in different connection lists

Commit:
LRAC-12905 Add CanSortProductListByAttribute testcase

Commit:
LRAC-12905 Update sort tests to check attribute position after sorting

Commit:
LRAC-12905 SF

manuferreira committed:
LRAC-12835 Add CanSelectAccountAttributes testcase

manuferreira committed:
LRAC-12835 Add CheckNotPossibleDeselectRequiredAttributesForAccounts testcase

manuferreira committed:
LRAC-12835 Add CanDeselectNonRequiredAccountAttributes testcase

manuferreira committed:
LRAC-12835 Add CheckAllDefaultsAttributesAreBeingCheckedForAccounts testcase

manuferreira committed:
LRAC-12835 SF

calvinctvo committed:
LRAC-12911 Add AllOrderAttributesRequired testcase

calvinctvo committed:
LRAC-12911 Add CheckNotPossibleToDeselectRequiredAttributesForOrders test case

calvinctvo committed:
LRAC-12911 Add CanSearchAttributeInOrdersList test case

calvinctvo committed:
LRAC-12911 Add CanPaginateInOrdersAttributeList test case

calvinctvo committed:
LRAC-12911 Add CanSortOrdersListByAttribute test case

calvinctvo committed:
LRAC-12911 SF

manuferreira committed:
LRAC-12911 Update story tickets to match the actual one

manuferreira committed:
LRAC-12911 Update tests to not convert the string to a list since its already converted when using a for loop

JhossephG committed:
LPS-175309 Add deleteAllCommerceOrdersAndSitesViaAPI macro for our tearDown

danielCamara02 committed:
LPS-174896 Add CanDeletePortletModifiedConfigurationWithoutDeletionModificationConflict test

p2kmgcl committed:
LPS-176789 Add tsconfig.json

p2kmgcl committed:
LPS-176789 Migrate constants to TypeScript

p2kmgcl committed:
LPS-176789 Migrate DateTimeInput to TypeScript

p2kmgcl committed:
LPS-176789 Add range property to support date ranges

marcogalluzzi committed:
LPS-175062 Avoid opening suggestion pane

p2kmgcl committed:
LPS-176789 Use a state ref instead of usePrevious

usePrevious hook initiates it's internal ref as undefined,
which makes sense because there is no "previous" ref when
the component has been mounted.

However, we want to keep the initial step as previous value,
indicating that there have been no changes in the input.

p2kmgcl committed:
LPS-176789 Remove previousValueRef.current check

We now for sure that, if value is a DateRange,
previousValueRef.current will have a range too.

p2kmgcl committed:
LPS-176789 Add new test with date ranges

carolmariaabb committed:
LPS-176235 Simplify code

carolmariaabb committed:
LPS-176235 Move upgrade to service and use SQL syntax instead

Commit:
LPS-176055 Remove .management-bar-default CSS class

p2kmgcl committed:
LPS-176789 Rename input/output format to internal/display

Commit:
LPS-173135 batch-planner - hide CSV support behind feature flag

Commit:
LPS-173135 batch-planner - JSP - hide CSV behind feature flag

Commit:
LPS-173135 batch-planner - JS - hide CSV behind feature flag

p2kmgcl committed:
LPS-176789 Extract utility functions

manuferreira committed:
LRAC-12911 Update the testcases related to people attributes because of a fix made in LRAC-13046

Commit:
LRAC-13052 Move these tests to SyncContactsAndAccounts.testcase and rename

Commit:
LRAC-13052 Update test to add steps to verify if the users appear in AC

Commit:
LRAC-13052 Remove the test which is covered by CanSyncAllContacts testcase

Commit:
LRAC-13052 Remove the test that will no longer be used in the new wizard

Commit:
LRAC-13052 Modify the setUp steps to work with all tests in that file

Commit:
LRAC-13052 Remove the use of refresh to exit the wizard

Commit:
LRAC-13052 SF

davisaints committed:
LPS-175197 Add CanCreateActionNotificationUsingOldValueWithOnAfterAddTrigger test

Commit:
LPS-173135 portal properties - disable feature flag

adolfopa committed:
LPS-132310 Use same message as other previewers for consistency

adolfopa committed:
LPS-132310 Build lang

Commit:
LPS-177321 Refactoring throughout the EVP project to standardize folders and files

adolfopa committed:
LPS-176576 Use folder ID, not site ID

Commit:
COMMERCE-9634 Increase base stock for products with 0 stock on Minium and Speedwell Initializers

Commit:
COMMERCE-9634 Check if product has stock before disabling back orders

davisaints committed:
LPS-175196 Update macro to remove the save step

davisaints committed:
LPS-175196 Add CanCreateValidationUsingOldValueFunctionWithPicklistField test

pyoo47 committed:
LRCI-3478 Extend timeout to 3 hours

thaiscabrals committed:
LPS-176309 - Fix DataEngineRenderer#TEXT_INPUT_INDEXED path

petershin committed:
LPS-176938 Partial revert

thaiscabrals committed:
LPS-176314 - Fix DataEngineRenderer.path

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

JhossephG committed:
LPS-177372 Fix CanCreateActionNotificationUsingOldValueWithOnAfterUpdateTrigger Test

Kvale1733 committed:
LRQA-79702 Add test to acceptance and raise priority

gamerson committed:
LPS-177378 check if the from in assemble block is a taskName

Hanlf committed:
LPS-177383 Assert the document file name is updated

Commit:
LPS-174896 Use the existing macro to instead of the steps

Commit:
LPS-171119 View template and create publication with it after upgrading

ling-alan-huang committed:
LPS-168272 SF

brianwulbern committed:
LRQA-51488 Fix typo

brianwulbern committed:
LRQA-51488 Regen

liulukiwi committed:
LPS-177287 Modify cases to add navigation menu by clicking Add button

liulukiwi committed:
LPS-177391 Modify macro to select asset in mapped URL

sandrodw3 committed:
LPS-176065 Restore status label in asset publisher

sandrodw3 committed:
LPS-176065 Restore status label in asset list

liferay-translation committed:
LPS-77699 Update Translations

Jeremy-Ch committed:
LPS-132310 Update the generation info

brianchandotcom committed:
LPS-177378 prep next

brianchandotcom committed:
LRCI-3478 prep next

Commit:
COMMERCE-9634 Show Exception in UI when disabling Back Orders with empty stock

Commit:
COMMERCE-9634 Introduce new language key

Commit:
COMMERCE-9634 buildLang
